vitrina/cms/templates/pages/page_with_side_menu.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+{% load cms_tags sekizai_tags %}
+{% load menu_tags %}
+{% load static %}
+{% load sass_tags %}
+<html>
+<head>
+    <meta charset="utf-8">
+
+    <title>Lietuvos atvirų duomenų portalas{% block pageTitle %}{% endblock %}</title>
+    <meta name="description" content="{% block pageDescription %}Atvirų duomenų portalas - tai vieningas prieigos taškas prie visų Lietuvoje atvertų duomenų rinkinių ir suteikiantis technologines priemones atvirų duomenų tiekėjams parengti ir publikuoti atvertų duomenų rinkinių metaduomenis.{% endblock %}">
+    <meta name="keywords" content="{% block pageKeywords %}atviri duomenys, duomenų rinkiniai, atvirų duomenų portalas{% endblock %}">
+    <meta name="author" content="{% block pageAuthor %}Informacinės visuomenės plėtros komitetas{% endblock %}">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <meta property="og:title" content="Lietuvos atvirų duomenų portalas{% block pageOgTitle %}{% endblock %}" />
+    <meta property="og:type" content="{% block pageOgType %}website{% endblock %}" />
+    <meta property="og:description" content="{% block pageOgDescription %}Atvirų duomenų portalas - tai vieningas prieigos taškas prie visų Lietuvoje atvertų duomenų rinkinių ir suteikiantis technologines priemones atvirų duomenų tiekėjams parengti ir publikuoti atvertų duomenų rinkinių metaduomenis.{% endblock %}" />
+
+    <link type="text/css" href="{% static 'css/bundle.css' %}" rel="stylesheet" media="screen">
+    <script defer src="{% static 'js/bundle.js' %}"></script>
+    <script src="https://code.jquery.com/jquery-3.6.0.js"></script>
+    {% include "component/favicon.html" %}
+    {% render_block "css" %}
+</head>
```

Comment:
Use `{% extends "base.html" %}` and move things like cms toolbar to base.html. This template should only add a sidebar.